app/helpers/curation_concerns/collections_helper.rb

```diff
@@ -17,8 +17,9 @@ def link_to_select_collection(collectible, opts = {})
 
   # override hydra-collections
   def link_to_remove_from_collection(document, label = 'Remove From Collection')
-    link_to collections.collection_path(@collection.id, collection: { members: 'remove' },
-                                                        batch_document_ids: [document.id]), method: :put do
+    collection_id = @collection ? @collection.id : @presenter.id
```

Comment:
@tampakis I see @presenter.id here and below I see @presenter.collection.  I'm wondering is this the same presenter?  @presenter.collection makes more sense to me.